queries/roc/locals.scm

```diff
@@ -0,0 +1,40 @@
+(expr_body) @scope
```

Comment:
Please follow upstream treesitter locals naming, and CONTRIBUTING.md guidelines

Reply:
I have changed the naming, please let me know if this is correct.